packages/editor-ui/src/components/NodeCredentialType.vue

```diff
@@ -0,0 +1,151 @@
+<template>
+	<div>
+		<div class="parameter-value-container">
+			<n8n-select
+				:size="inputSize"
+				filterable
+				:value="displayValue"
+				:placeholder="parameter.placeholder ? getPlaceholder() : $locale.baseText('parameterInput.select')"
+				:loading="remoteParameterOptionsLoading"
+				:disabled="isReadOnly || remoteParameterOptionsLoading"
+				:title="displayTitle"
+				@change="valueChanged"
+				@keydown.stop
+				@focus="setFocus"
+				@blur="onBlur"
+			>
+				<n8n-option
+					v-for="credType in supportedCredentialTypes"
+					:value="credType.name"
+					:key="credType.name"
+				>
+					<div class="list-option">
+						<div class="option-headline">
+							{{ credType.displayName }}
+						</div>
+						<div
+							v-if="credType.description"
+							class="option-description"
+							v-html="credType.description"
+						/>
+					</div>
+				</n8n-option>
+			</n8n-select>
+			<slot name="issues-and-options" />
+		</div>
+
+		<scopes-notice
+			v-if="activeCredential.scopes.length > 0"
+			:scopes="activeCredential.scopes"
+			:shortCredentialDisplayName="activeCredential.shortDisplayName"
+		/>
+		<div>
+			<node-credentials
+				:node="node"
+				:overrideCredType="node.parameters.nodeCredentialType"
+				@credentialSelected="credentialSelected"
+			/>
+		</div>
+	</div>
+</template>
+
+<script lang="ts">
+import { ICredentialType } from 'n8n-workflow';
+import Vue from 'vue';
+import { mapGetters } from 'vuex';
+import ScopesNotice from '@/components/ScopesNotice.vue';
+import NodeCredentials from '@/components/NodeCredentials.vue';
+
+export default Vue.extend({
+	name: 'NodeCredentialType',
+	components: {
+		ScopesNotice,
+		NodeCredentials,
+	},
+	props: [
+		'node',
+		'credentialSelected',
+		'parameter',
+		'activeCredential',
+		'inputSize',
+		'displayValue',
+		'remoteParameterOptionsLoading',
```

Comment:
not sure this is needed here.. because we load all credential types as soon as instance loads